app/assets/templates/poll_tpl.jst.hbs

```diff
@@ -13,15 +13,26 @@
       {{#poll.poll_answers}}
         <div class="result-row">
           <input type="radio" name="vote" value="{{id}}"/>
-          <label>{{answer}}</label>
-          <div class="poll-result pull-right">
-            <span class="percentage"></span>
-            ({{t "poll.answer_count" count=vote_count}})
-          </div>
-          <div class="poll_progress_bar_wrapper progress">
+          {{#if isCurrentUserVote}}
+            <label>
+            {{answer}}
```

Comment:
I'm sorry if I made a mistake in my comment before (I'm unable to check right now) but here are two whitespaces missing for indentation.